### ISSUE TITLE
Message Load Generator

### ISSUE THREAD

**Issue by araskachoi** [whiteblock/gossipsub-testing#9]
have a separate generator that will send messages
- pass arguments for:
  - int: `number_of_messages_per_second`
  - arr: `ip_to_send_to`

**Comment by araskachoi** [comment]
@adam-hanna
any comments on this? I think it will be important to separate the logic of message sending so that it is external to the client. This way we can keep the libp2p client will solely use their computational resources to relay messages, instead of sending them.

Let me know if you need more clarifications on this or have any questions!

**Comment by adam-hanna** [comment]
Yup, this has been implemented in `cmd/client`

Messages per second hasn't been implemented, but could be done by wrapping in a shell script or something

**Comment by araskachoi** [comment]
So the implementation that you made is inside the client. In your opinion, also might need to get feedback from PL, do you think that this is the best way to implement the message load generator? This would mean that the container that is running the client will also be responsible for sending the messages. This might cause the container to have more overhead than needed? I think it would be best to separate the logic so that we can have unbiased test results. what do you think?

**Comment by araskachoi** [comment]
Right, so the best way to do this would to have the client implementation send these messages. basically load the `host` with messages that are made inside `client` and pass them via RPC at a specified rate
